modules/network-observability-total-resource-usage.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+// * network_observability/configuring_operator.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="network-observability-total-resource-usage-table_{context}"]
+= Total average memory and CPU usage
+The following table outlines averages of total resource usage for clusters with a sampling sizes of 1, 50, and 400.
+
+[IMPORTANT]
+====
+The examples outlined in the table demonstrate scenarios that are tailored to specific workloads. Consider each example only as a baseline from which adjustments can be made to accommodate your workload needs.
+====
+
+.Total average resource usage
+[options="header"]
+|===
+| Sample size | Parameters | Small (25 nodes)  | Medium (65 nodes) ^[2]^ | Large (120 nodes) ^[2]^
+.6+| *Sampling = 1* | *With Loki* 3+|
+| Total NetObserv CPU Usage | 3.24 | 3.42 | 7.30
+| Total NetObserv RSS (Memory) Usage | 14.09 GB | 11.81 GB | 36.46 GB
+| *Without Loki* 3+|
+| Total NetObserv CPU Usage | 2.40 | 2.43 | 5.59
+| Total NetObserv RSS (Memory) Usage | 6.85 GB | 10.39 GB | 13.92 GB
+.6+| *Sampling = 50* | *With Loki* 3+|
+| Total NetObserv CPU Usage | 2.04 | 2.36 | 3.31
+| Total NetObserv RSS (Memory) Usage | 8.79 GB | 19.14 GB | 21.07 GB
+| *Without Loki* 3+|
+| Total NetObserv CPU Usage | 1.55 | 1.64 | 2.70
+| Total NetObserv RSS (Memory) Usage | 6.71 GB | 10.15 GB | 14.82 GB
+.6+| *Sampling = 400* | *With Loki* 3+|
+| Total NetObserv CPU Usage | 1.71 | 1.44 | 2.36
+| Total NetObserv RSS (Memory) Usage | 8.21 GB | 16.02 GB | 17.44 GB
+| *Without Loki* 3+|
+| Total NetObserv CPU Usage | 1.31 | 1.06 | 1.83
+| Total NetObserv RSS (Memory) Usage | 7.01 GB | 10.70 GB | 13.26 GB
+|===
+[.small]
+--
+1. Tested with AWS M6i instances.
+2. In addition to this worker and its controller, 3 infra nodes (size `M6i.12xlarge`) and 1 workload node (size `M6i.8xlarge`) were tested.
+--
+Summary: Average total resource usage of Network Observability (Agents+FLP+Kafka+Loki), for a 120 nodes cluster with an average traffic of 2.5 GBps, configured with sampling 1, is: 7.4 CPU cores (out of 1920), 39.82 GB. It generated an average of 17K flows per second.
```

Comment:
This sentence doesn't correspond to the numbers presented above. We can either adapt it, or completely remove it?

Reply:
@jotak @memodi Thanks for the clarification! Since the table is more comprehensive than just sampling value of 1, the summary now provides a generic statement that this table only presents the average total resource usage of Network Observability (Agents+FLP+Kafka+Loki). 

CC: @skrthomas :)